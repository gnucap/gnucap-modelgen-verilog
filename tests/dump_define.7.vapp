
`define thing(a,b) module a \
real b; \
endmodule

`thing(A,B)

`define TMPHICJ(c_j,vd0,z,w,is_al,vgeff,c_j_t,vd_t,w_t) \
        vdj0    = 2*vt0*ln(exp(vd0*0.5/vt0)-exp(-0.5*vd0/vt0));

module test()

	begin analog
		real a,b,c,d,e,f,g,h,i;
		real vdj0, vdjt, vdt;
`TMPHICJ(a,b,c,d,e,f,g,h,i)
	end

endmodule
