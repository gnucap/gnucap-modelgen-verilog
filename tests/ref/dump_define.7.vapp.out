

 module A real B; endmodule 


module test()

	begin analog
		real a,b,c,d,e,f,g,h,i;
		real vdj0, vdjt, vdt;
         vdj0    = 2*vt0*ln(exp(b*0.5/vt0)-exp(-0.5*b/vt0)); 
	end

endmodule
